### PR TITLE
`spinner` should not come up in non-interactive query mode. Closes #227

### LINKED ISSUE #227: `spinner` should not come up in non-interactive query mode

### PR DESCRIPTION
**Pull request by binaek89**

### CODE REVIEW

File: db/interactive_client.go [hunk 1]
```diff
@@ -205,7 +205,7 @@ func (c *InteractiveClient) executor(line string, resultsStreamer *ResultStreame
 		resultsStreamer.Done()
 	} else {
 		// otherwise execute query
-		if result, err := c.client.executeQuery(query, true, cmdconfig.Viper().Get(constants.ArgOutput) == constants.ArgTable); err != nil {
+		if result, err := c.client.executeQuery(query, cmdconfig.Viper().Get(constants.ArgOutput) == constants.ArgTable); err != nil {
```
Comment (kaidaguerre) [inline db/interactive_client.go:208]: put `cmdconfig.Viper().Get(constants.ArgOutput) == constants.ArgTable)` into a variable

**Review by kaidaguerre — CHANGES_REQUESTED** [review]

File: cmd/query.go [hunk 2]
```diff
@@ -71,6 +73,13 @@ func runQueryCmd(cmd *cobra.Command, args []string) {
 
 	queryString := getQuery(cmd, args)
 
+	// set the flag to not show spinner
+	cmdconfig.Viper().Set("show-spinner", false)
```
Comment (kaidaguerre) [inline cmd/query.go:77]:
```
showSpinner := queryString == ""
cmdconfig.Viper().Set("show-spinner", showSpinner)
```

File: cmdconfig/viper.go [hunk 3]
```diff
@@ -28,14 +28,17 @@ func InitViper(v *viper.Viper) {
 			utils.FailOnError(err)
 		}
 	}
+
+	// set defaults
+	v.Set("show-spinner", true)
```
Comment (kaidaguerre) [inline cmdconfig/viper.go:33]: so we ONLY disable spinner for non interactive queries?

**Review by kaidaguerre — CHANGES_REQUESTED** [review]

File: cmdconfig/viper.go (hunk 3 above)
Comment (binaek89) [reply cmdconfig/viper.go:33]: yes.

otherwise for things like `service start` and `service refresh`, the spinner will stop working

**Pull request merged**